app/components/settings/billing-page/renewal-section.hbs

```diff
@@ -0,0 +1,12 @@
+<Settings::FormSection @title="Renewal" @description="Details of your upcoming payment">
+  <div class="inline-flex items-center gap-2 rounded-full bg-gray-50 dark:bg-gray-800 px-2 py-1 border border-gray-300 dark:border-gray-600">
```

Comment:
@Arpan-206 is this the same as `StatusPill` with a "gray" color? If so let's re-use that here. You can move it to `Settings::BillingPage::StatusPill` to make it clear multiple components share it